synapse_data_warehouse/synapse/dynamic_tables/V2.28.5__add_comments_certifiedquizquestion_latest.sql

```diff
@@ -0,0 +1,16 @@
+-- Configure environment
+USE SCHEMA {{database_name}}.synapse; --noqa: JJ01,PRS,TMP,CP02
+
+-- Add the table and column comments
+ALTER DYNAMIC TABLE CERTIFIEDQUIZQUESTION_LATEST SET COMMENT = 'This table contains the latest snapshots of the questions of the certification quiz. With each entry representing a question answered by the user during the quiz.';
```

Comment:
Nit: it could be more informative if we add time window here.

Reply:
[Updated](https://github.com/Sage-Bionetworks/snowflake/pull/99/commits/91ab11feb51d7d3a65142c2023aa2d1e63681907) ✔️